ci/tag-docs.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+### This script should only be run on Improbable's internal build machines.
+### If you don't work at Improbable, this may be interesting as a guide to what software versions we use for our
+### automation, but not much more than that.
+
+set -e -u -x -o pipefail
+
+if [[ -z "$BUILDKITE" ]]; then
+  echo "This script is only to be run on Improbable CI."
+  exit 1
+fi
+
+cd "$(dirname "$0")/../"
+
+source ci/tools.sh
+
+# Tag a version of the docs.
+#
+# Environment variables as arguments:
+#   DOCS_TYPE: "unity"
+#   DOCS_VERSION: "alpha"
+#   DOCS_TARGET: "testing", "staging" or "production"
+
+DOCS_HASH="$(buildkite-agent meta-data get "docs-hash")"
+if [ -z "${DOCS_HASH}" ]; then
+    echo "Expected docs-hash meta-data to be set."
```

Comment:
Won't the `buildkite-agent` command will fail before we get here?

Reply:
yes - removing